build.gradle

```diff
@@ -38,14 +38,10 @@ subprojects {
     configurations.all {
         resolutionStrategy {
             failOnVersionConflict()
-            // Force the IDEA SDK provided version of these dependencies to avoid classloading issues.
-            force 'org.apache.httpcomponents:httpclient:4.4.1', 'com.google.guava:guava:17.0'
```

Comment:
so i guess you were able to verify that the failOnVersionConflict wasn't catching the idea provided libs?

If so, we should file an issue with the gradle-intellij-plugin